docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,187 @@
+        - go to rule assertion: `GO TO RULE ASSERTION`
+ 
+    - otherwise, the capacity per volume is not  < 4000 btu/ gallon: `else:`
+      - indicate with the note that the capacity per volume exceeds the limit: `rule_note += "  Capacity per volume exceeds the limit of 4000 btu/hr/gallon given for Electric Storage Water Heaters in ASHRAE 90.1 Table 7.8."`
+      - set the rule status to FAIL and continue to rule assertion: `rule_status = "FAIL"; CONTINUE TO RULE ASSERTION`
+
+    - else if the storage volume is between 2 and 20 gallons, in which case there is no rating in APPENDIX F.  set rule status to UNDETERMINED: `elif storage_volume < 20:`
+      - set the rule status to UNDETERMINED: `rule_status = "UNDETERMINED"`
+      - add a note: `rule_note += "  The storage tank volume is between 2 and 20 gallons and based on the size categories in ASHRAE 90.1 Table 7.8 and 10 CFR 430 it would not be classified as an electric storage water heater. Consequently, this rule was not assessed for this service water heater.  Note that the specific requirements of 10 CFR 430 can be found in ASHRAE 90.1 Appendix F Table F-2."`
+      - go to rule assertion: `CONTINUE TO RULE ASSERTION`
+
+        
+  - otherwise, this is an electric resistance water heater with input > 12 kW, continue with logic per Table 7.8: `else:`
+    - check electric storage water heaters with capacity greater than 12 kW: `if swh_type == "ELECTRIC_RESISTANCE_STORAGE" and input_capacity > 40945.7 (btu/hr):`
+      - check that the capacity_per_volume is < 4000 btu/hr/gallon: `if capacity_per_volume < 4000:`
+        - calculate the standby loss target: `standby_loss_target = 0.3 + (27/swh_eq.distribution_system.tank.storage_volume)`
+        - check whether the standby loss fraction is given, if it's not given, provide a note, and a rule result of UNDETERMINED: `if swh_eq.standby_loss_fraction == NULL: rule_note += "  Electric resistance water heaters with input capacity > 12kW have a required standby loss given by ASHRAE 90.1 Table 7.8.  No standby loss was given for this water heater, thus the rule was not able to be evaluated."`
+        - if the standby_loss is equal to the target, set the rule_status to PASS: `if swh_eq.standby_loss_fraction <= standby_loss_target: rule_status = "PASS"`
+        - otherwise: `else:`
+          - set the note to indicate that the standby_loss is higher than the maximum allowed: `rule_note += "  Standby Loss is higher than required by Table 7.8"; rule_status = FAIL; GO TO RULE ASSERTION`
+      - otherwise: `else:`
+        - set the note to indicate that the capacity per volume exceeds the limit: `rule_note += "  Capacity per volume exceeds the limit of 4000 btu/hr/gallon given for Electric Storage Water Heaters in ASHRAE 90.1 Table 7.8."; rule_status = FAIL; GO TO RULE ASSERTION` 
+    - check if the water heater is gas storage: `if swh_type == "GAS_STORAGE":`
```

Comment:
this check is inside of the branch where: "this is an electric resistance water heater with input > 12 kW"
`